flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/rest/SqlGatewayRestEndpointITCase.java

```diff
@@ -532,27 +468,30 @@ public EmptyMessageParameters getUnresolvedMessageParameters() {
         }
     }
 
-    private static class TestVersionSelectionHeaders1 extends TestVersionSelectionHeadersBase {
+    private static class TestVersionSelectionHeaders0 extends TestVersionSelectionHeadersBase {
         @Override
         public Collection<SqlGatewayRestAPIVersion> getSupportedAPIVersions() {
             return Collections.singleton(SqlGatewayRestAPIVersion.V0);
         }
     }
 
-    private static class TestVersionSelectionHeaders2 extends TestVersionSelectionHeadersBase {
+    private static class TestVersionSelectionHeaders12 extends TestVersionSelectionHeadersBase {
         @Override
         public Collection<SqlGatewayRestAPIVersion> getSupportedAPIVersions() {
-            return Collections.singleton(SqlGatewayRestAPIVersion.V1);
+            return Arrays.asList(SqlGatewayRestAPIVersion.V1, SqlGatewayRestAPIVersion.V2);
```

Comment:
```
 List<SqlGatewayRestAPIVersion> versions =
                    Arrays.asList(SqlGatewayRestAPIVersion.values());
            versions.remove(SqlGatewayRestAPIVersion.V0);
            return versions;
```

Reply:
Can't remove or add value to the list returned by `Arrays.asList.` I used `new ArrayList(Arrays.asList(xx))`.